drivers/flash/spi_nor.h

```diff
@@ -42,11 +42,57 @@
 #define SPI_NOR_CMD_4BA         0xB7    /* Enter 4-Byte Address Mode */
 #define SPI_NOR_CMD_DPD         0xB9    /* Deep Power Down */
 #define SPI_NOR_CMD_RDPD        0xAB    /* Release from Deep Power Down */
+#define SPI_NOR_CMD_WR_CFGREG2  0x72    /* Write config register 2 */
+#define SPI_NOR_CMD_RD_CFGREG2  0x71    /* Read config register 2 */
+#define SPI_NOR_CMD_RESET_EN    0x66    /* Reset Enable */
+#define SPI_NOR_CMD_RESET_MEM   0x99    /* Reset Memory */
+#define SPI_NOR_CMD_BULKE       0x60    /* Bulk Erase */
+#define SPI_NOR_CMD_PP_4B       0x12    /* Page Program 4 Byte Address */
+#define SPI_NOR_CMD_READ_FAST_4B  0x0C  /* Fast Read 4 Byte Address */
```

Comment:
LGTM

Not a showstopper, but would be great to align the define values under `/* Flash opcodes */`.

Reply:
will be updated in the next PR